`Alert` and `Toast` Overview Update (#1720)
hashicorp/design-system: website/docs/components/toast/partials/guidelines/overview.md

> Toasts deliver messages to communicate an ongoing process or result of a user action. They typically disappear after a set interval or upon user dismissal, offering a non-intrusive way to provide feedback. 

Reviewer: Slight change to communicate that toasts may not always be the right component to communicate a result of a user action. wdyt?

```suggestion
Toasts deliver messages to communicate an ongoing process or, sometimes, a result of a user action. They typically disappear after a set interval or upon user dismissal, offering a non-intrusive way to provide feedback. 
```

> Toasts deliver messages to communicate an ongoing process or as a result of a user's actions. They typically disappear after a set interval or upon user dismissal, offering a non-intrusive way to provide feedback. 

Reviewer 2: ```suggestion
Toasts deliver messages to communicate an ongoing process or result of a user action. They typically disappear after a set interval or upon user dismissal, offering a non-intrusive way to provide feedback. 
```

> Toasts deliver messages resulting from a user's actions. They typically disappear after a set interval or upon user dismissal, offering a non-intrusive way to provide feedback. While [Alerts](/components/alert) handle system-generated messages or critical information, Toasts suit less urgent feedback, such as indicating a user-initiated process's success or ongoing status.

Reviewer: Providing feedback based on our earlier conversation. I removed the second half of this section to:

1. reduce confusion around terms such as "system-generated"
2. remove duplicative information (info is also presented in the when to use/when not to use section)

```suggestion
Toasts deliver messages to communicate an ongoing process or as a result of a user's actions. They typically disappear after a set interval or upon user dismissal, offering a non-intrusive way to provide feedback. 
```
